src/labels/CustomLabels.labels

```diff
@@ -6631,6 +6647,78 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Message when field/Field Mapping not found and user might not have access to it.</shortDescription>
         <value>This form contains fields that can't be found. Please check with your administrator.</value>
     </labels>
+    <labels>
+        <fullName>gsAdminSetupTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title in Get Started</shortDescription>
+        <value>ADMIN SETUP CHECKLIST</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistArrowDownAltText</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Arrow Icon alter text to expand state</shortDescription>
+        <value>Click to expand contents</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistArrowRightAltText</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Arrow Icon alter text to collapse state</shortDescription>
```

Comment:
Minor change. Should say either "Arrow icon alternative text to collapse state" or "Arrow icon alt text to collapse state".